website/client/components/guilds/discovery.vue

```diff
@@ -0,0 +1,98 @@
+<template lang="pug">
+.row
+  sidebar(@search="updateSearch", @filter="updateFilters")
+
+  .col-10
+    .clearfix
+        h1.page-header.float-left(v-once) {{ $t('publicGuilds') }}
+        b-form-select.float-right.sort-select(v-model='sort', :options='sortOptions')
```

Comment:
this should use the dropdown not the select, see the equipment page for the code (with a Sort label next to the dropdown)